patches/components-optimization_guide-core-BUILD.gn.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/components/optimization_guide/core/BUILD.gn b/components/optimization_guide/core/BUILD.gn
+index 29444c9b70244c041e3522fbff0b8eeaf99d55a5..da432afd5be59e023caedfa2e8d0e3ed1b63f3bb 100644
+--- a/components/optimization_guide/core/BUILD.gn
++++ b/components/optimization_guide/core/BUILD.gn
+@@ -65,6 +65,14 @@ static_library("model_executor") {
+     "//components/optimization_guide:machine_learning_tflite_buildflags",
+     "//third_party/re2",
+   ]
++  deps = [
++    ":features",
++    "//base",
++    "//components/optimization_guide/proto:optimization_guide_proto",
++    "//net",
++    "//url",
++  ]
++
+   if (build_with_tflite_lib) {
+     public_deps += [
+       "//components/optimization_guide/core:machine_learning",
+@@ -74,15 +82,8 @@ static_library("model_executor") {
+       "//third_party/tflite_support",
+       "//third_party/tflite_support:tflite_support_proto",
+     ]
++    deps += [ ":machine_learning" ]
```

Comment:
this is upstream patch. https://chromium-review.googlesource.com/c/chromium/src/+/4031249